lib/parsers/webhook_json_parser.rb

```diff
@@ -100,13 +101,11 @@ def deleted?
       end
 
       def repo_name
-        case @vcs
-        when 'github'
-          @data['repository']['name']
-        when 'gitlab'
+        if @vcs == 'gitlab'
           @data['project']['name']
+        else
+          @data['repository']['name']
```

Comment:
or to be honest just an end of line comment confirming that @data['repository']['name'] is used by github. stash, bitbucket and tfs.  Probably not an issue at all if we start testing this class against some sample payloads.

Reply:
oops, have I just accidentally mentioned a random github user? SORRY!!!